### PR TITLE
refactor: move changelogPath down to ReleasePR

### PR DESCRIPTION
**Pull request by joeldodge79**
The releasers already know where their changelog is. It should be
configurable for the `release-pr` command as well.

Also cleaned up test/releasers/python.ts a bit and tested the custom and
default changelogPath settings there.

### CODE REVIEW

File: test/factory.ts [hunk 1]
```diff
@@ -159,49 +196,55 @@ describe('factory', () => {
...
+        repoUrl: 'googleapis/ruby-test-repo',
         defaultBranch: '1.x',
         fork: true,
         token: 'some-token',
         apiUrl: 'https://some.api.com',
         releaseType: 'ruby',
         label: 'foo,bar',
         path: 'some/path',
-        packageName: 'simple-test-repo',
+        packageName: 'ruby-test-repo',
```
Comment (joeldodge79) [inline test/factory.ts:233]: cosmetic changes for consistency with `releastType: 'ruby'`

**Comment by joeldodge79** [comment]
hm, interesting that node10 ci job failed (but 12 and windows passed). Also, that codecov/proj really chaps my hide - not sure why my PR is getting dinged for coverage drops on all those releasers. Gonna game it and add a quick test for the `Ruby` releaser ;-)

**Comment by joeldodge79** [comment]
node 10 doesn't like the `creates a release PR with custom config` test/snapshot. I'll dig into why that is...

File: __snapshots__/python.js [hunk 2]
```diff
@@ -204,7 +204,217 @@ fork: false
...
+
+### Killer Bug Fixes
+
+* **deps:** update dependency com.google.cloud:google-cloud-spanner to v1.50.0 ([08ca011](https://www.github.com/googleapis/py-test-repo/commit/08ca01180a91c0a1ba8992b491db9212))
+* **deps:** update dependency com.google.cloud:google-cloud-storage to v1.120.0 ([845db13](https://www.github.com/googleapis/py-test-repo/commit/845db1381b3d5d20151cad2588f85feb))
+
+
+### Sweet New Features
+
+* still no major version ([c0d3c41](https://www.github.com/googleapis/py-test-repo/commit/c0d3c4181ecf6c7337a39da1755805b2))
```
[the remark this answers is not in the document]
Comment (joeldodge79) [reply __snapshots__/python.js:232]: interesting, this is the output with node 12,14,windows but node 10 wants  to swap the order of chore/feat

**Comment by joeldodge79** [comment]
haha, now node10 passes but 12/14/windows fails :-) probably have to do some explicit ordering on `changelogSections`? Or is there a way to gate tests based on node version?

**Comment by joeldodge79** [comment]
I think #601 is still an issue - I had this for my changelogSections:
```
[
  {type: 'feat', section: 'Sweet New Features'},
  {type: 'fix', section: 'Killer Bug Fixes'},
  {type: 'chore', section: 'Do Your Chores'},
]
```

node 10 consistently orders it:
```
Sweet New Features
Killer Bug Fixes
Do Your Chores
```

but node 12,14,windows orders it:
```
Do Your Chores
Killer Bug Fixes
Sweet New Features
```

I'm going to remove the custom `changelogSections` snapshotting from my tests to unblock this PR but something still isn't right there...

**Comment by bcoe** [comment]
@joeldodge79 I've been noticing flip flopping sections when building  the action too. I'm betting it's something similar to what was addressed [here](https://github.com/conventional-changelog/conventional-changelog/commit/e0081a829133891e2def4a7b7ee5fa25f1440049#diff-0be7dd641bbcd1223c71f878bbf5e62b4402c98ff8e6d7d7bdd67edc3189e59e). Where the sorting helper in conventional-changelog was somehow non-deterministic on different Node.js versions.

I guess I'll need to dig into this again at some point.

**Review by bcoe — APPROVED** [review]

File: test/releasers/ruby.ts [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+      const releasePR = new Ruby({
+        github: new GitHub({owner: 'googleapis', repo: 'ruby-test-repo'}),
+        packageName: pkgName,
+        path: 'projects/ruby',
+        bumpMinorPreMajor: true,
+        monorepoTags: true,
+        changelogPath: 'HISTORY.md',
+      });
+
+      stubSuggesterWithSnapshot(sandbox, this.test!.fullTitle());
```
[the remark this answers is not in the document]
Comment (joeldodge79) [reply test/releasers/ruby.ts:107]: IIRC `this.test` is only defined when run with node. I guess you can execute this test suite in a browser too, in which case `this.test` is undefined apparently

**Pull request merged**